### DISCUSSION TITLE
Issue with espeak languages

### DISCUSSION THREAD

**Discussion by dostee-hemin** [coqui-ai/TTS#1137] [General Q&A] [answered]
I'm trying to train a TTS model on a language that is supposed to be supported by espeak (Kurdish "ku") using phonemes, but when I try to train the model the following error occurs:

File "c:\root\TTS\tts\utils\text\__init__.py", line 56, in text2phone
    raise ValueError(f" [!] Language {language} is not supported for phonemization.")
ValueError:  [!] Language ku is not supported for phonemization.

It says the Kurdish language is not supported for phonemization, but I have installed the latest version of espeak NG and it includes Kurdish. Am I missing something here?

To move past this issue, I ignored it entirely and instead decided to take a character-based training approach rather than phoneme-based. In other words, "use_phonemes = False". What is the difference between character-based training and phoneme-based training? Will my model sound different if I pick one over the other?

**Comment by erogol** [comment, marked as answer]
Current version does not use espeak. It uses Gruut. However we are working on enabling espeak.

Until then you can train with graphemes. It should work fine for Kurdish.

**Reply by dostee-hemin** [reply]
Aha okay. Does this mean I need to set "use_phonemes" to True or is there some other configuration I need to perform specifically for graphemes?

**Reply by erogol** [reply]
If you want to train with graphemes you just need to set use_phonemes false

**Reply by tansilandre** [reply]
hi @erogol is there any way we can use espeak? thanks in advance!

**Reply by manmaynakhashi** [reply]
@tansilandre it's been integrated i am using it.

**Reply by tansilandre** [reply]
@manmaynakhashi how to do that? or how is the configuration you use?

**Reply by manmay-nakhashi** [reply]
@tansilandre 
```    text_cleaner="hindi_cleaners",
    use_phonemes=True,
    phonemizer="espeak",
    phoneme_language="hi",
    phoneme_cache_path=os.path.join(output_path, "phoneme_cache"),
```
add this to your config
install espeak from apt-get

**Reply by tansilandre** [reply]
@manmay-nakhashi thank you!!

**Reply by naveed81** [reply]
How to use espeak-ng as phonemizer instead of espeak?
